README.md

```diff
@@ -2,30 +2,29 @@
 
 
 ## Overview
+OpenFin Notifications provides developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
 
-OpenFin Notifications provide developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
-
-Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the system tray notification icon.
+Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the Notifications icon in the system tray.
 
 This project consist of 3 parts:
-1. The Notifications Service, displaying and managing notifications and owning the Notification Center UI
-2. The Notifications Client, exposing API's for applications to create and manage notifications
+1. The Notifications Provider, displaying and managing notifications, and owning the Notification Center
```

Comment:
Related to a comment below, but you've decided that "Notifications service" is lower case, but have "Notifications Provider" here as upper case. Not saying the necessarily need to be the same, but I would say it's either both lower case or provider lower, Service upper. 

Open to comments from others on this too of course